src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ObjectExplorerService.cs

```diff
@@ -105,7 +98,8 @@ internal IReadOnlyCollection<string> SessionIds
         public override void SetServiceProvider(IMultiServiceProvider provider)
         {
             Validate.IsNotNull(nameof(provider), provider);
-            serviceProvider = provider;
+            var querierAssembly = typeof(SmoQuerier).Assembly;
+            serviceProvider = ExtensionServiceProvider.CreateFromAssembliesInDirectory( Path.GetDirectoryName(querierAssembly.Location), new string[] { Path.GetFileName(querierAssembly.Location)});

```

Comment:
What's going on here? You get passed in a provider, but then only use it once below? What about adding the new DLL to the default inclusion list instead so we don't have multiple different providers floating around?

https://github.com/microsoft/sqltoolsservice/blob/a160f382ff516144db2a8a95fa95e487e84d0a4a/src/Microsoft.SqlTools.Hosting/Extensibility/ExtensionServiceProvider.cs#L24